mobile/api_discovery.go

```diff
@@ -8,6 +8,12 @@ import (
 	"github.com/waku-org/go-waku/waku/v2/dnsdisc"
 )
 
+type DnsDiscoveryItem struct {
+	PeerID    string   `json:"peerID,omitempty"`
+	Addresses []string `json:"multiaddrs,omitempty"`
+	ENR       string   `json:"enr,omitempty"`
```

Comment:
does the the peerid and enr will always be present, or they may be missing? I'm confused of the `omitempty` tag.